README.md

```diff
@@ -52,84 +66,143 @@ pip install mcp-server-appwrite
 Then run the server using 
 
 ```bash
-python -m mcp_server_appwrite
+python -m mcp_server_appwrite [args]
 ```
 
+### Command-line arguments
+
+Both the `uv` and `pip` setup processes require certain arguments to enable MCP tools for various Appwrite APIs.
+
+> When an MCP tool is enabled, the tool's definition is passed to the LLM, using up tokens from the model's available context window. As a result, the effective context window is reduced.  
+>  
+> The default Appwrite MCP server ships with only the Databases tools (our most commonly used API) enabled to stay within these limits. Additional tools can be enabled by using the flags below.
+
+| Argument | Description |
+| --- | --- |
+| `--databases` | Enables the Databases API |
+| `--users` | Enables the Users API |
+| `--teams` | Enables the Teams API |
+| `--storage` | Enables the Storage API |
+| `--functions` | Enables the Functions API |
+| `--messaging` | Enables the Messaging API |
+| `--locale` | Enables the Locale API |
+| `--avatars` | Enables the Avatars API |
+| `--all` | Enables all Appwrite APIs |
+
 ## Usage with Claude Desktop
 
 Add this to your `claude_desktop_config.json`:
 
 ```json
-"mcpServers": {
-  "appwrite": {
-    "command": "uvx",
-    "args": [
-      "mcp-server-appwrite"
-    ],
-    "env": {
-      "APPWRITE_PROJECT_ID": "your-project-id",
-      "APPWRITE_API_KEY": "your-api-key",
-      "APPWRITE_ENDPOINT": "your-endpoint"  // Optional
+{
+  "mcpServers": {
+    "appwrite": {
+      "command": "uvx",
+      "args": [
+        "mcp-server-appwrite",
+        "--users"
+      ],
+      "env": {
+        "APPWRITE_PROJECT_ID": "your-project-id",
+        "APPWRITE_API_KEY": "your-api-key",
+        "APPWRITE_ENDPOINT": "https://cloud.appwrite.io/v1" // Optional
+      }
     }
   }
 }
+
```

Comment:
lets add a Note below that if you get the ENOENT uvx error, you need to either ensure uvx is added to the PATH, or use the full path to your uvx installation